<?xml version="1.0" encoding="UTF-8"?><testrun name="CoffeeMakerTest" project="CoffeeMaker" tests="6" started="6" failures="1" errors="0" ignored="0">
  <testsuite name="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.001">
    <testcase name="testAddRecipe1" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.001"/>
    <testcase name="testEditRecipe1" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.0"/>
    <testcase name="testPurchaseBeverage1" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.0"/>
    <testcase name="testDeleteRecipe1" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.0"/>
    <testcase name="testCheckInventory1" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.0">
      <failure>junit.framework.AssertionFailedError: expected:&lt;65&gt; but was:&lt;50&gt;
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at junit.framework.Assert.assertEquals(Assert.java:241)
	at junit.framework.TestCase.assertEquals(TestCase.java:409)
	at edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest.testCheckInventory1(CoffeeMakerTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at junit.framework.TestCase.runTest(TestCase.java:176)
	at junit.framework.TestCase.runBare(TestCase.java:141)
	at junit.framework.TestResult$1.protect(TestResult.java:122)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at junit.framework.TestResult.run(TestResult.java:125)
	at junit.framework.TestCase.run(TestCase.java:129)
	at junit.framework.TestSuite.runTest(TestSuite.java:252)
	at junit.framework.TestSuite.run(TestSuite.java:247)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</failure>
    </testcase>
    <testcase name="testAddInventory1" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.0"/>
  </testsuite>
</testrun>
